tekton/resources/cd/folder-template.yaml

```diff
@@ -75,18 +75,42 @@ spec:
             #!/bin/sh
             set -ex
```

Comment:
We *may* want to start migrating thoses to scripts in the repo ? (easier to change/debug, run independently)

Reply:
Yes, the are also downsides to doing that though.
The task will need a git repo to get the script from (I guess it could fetch it via Https too?) and the task and script version will be split. If we keep the task in here it's fine, but if we add it to the catalog, how would that work? Can we host scripts in the catalogue? Is there perhaps something we should add to the script mode? I.e. the ability to specify a script URL?

Reply:
Is this something we can sort out *after* this PR?

Reply:
@afrittoli of course :+1: it was a general "wondering" :stuck_out_tongue_closed_eyes: